test/JDBC/expected/BABEL-736-vu-verify.out

```diff
@@ -0,0 +1,14 @@
+SELECT degrees(a), degrees(b), degrees(c), degrees(d), degrees(e) from BABEL_736_vu_prepare_t_1 
```

Comment:
order by is mandatory for all the select statement returning multiple rows in test

Reply:
+1